### PR TITLE
Add min browser version error

### PR DESCRIPTION
**Pull request by nolway**
Add min version supported : 
Firefox : 80
Chrome: 80
Opera : 70
Safrari: 13

Screenshot from 2022-01-13 17-15-00

### CODE REVIEW

File: front/src/Utils/BrowserUtils.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        {
+            name: "chrome",
+            minVersion: 80,
+        },
+        {
+            name: "opera",
+            minVersion: 70,
+        },
+        {
+            name: "safrari",
```
Comment (moufmouf) [inline front/src/Utils/BrowserUtils.ts:26]: `safari` ? :D

**Review by moufmouf — CHANGES_REQUESTED** [review]
Just a small typo in the "safari" word.

Also, as an alternative to displaying a big error page, we already have a place in the application where we display error message for old browsers. Previously, we only checked that the WebRTC API was available, but we could require a more recent version here.

Check out `front/src/Stores/Errors/BrowserTooOldError.ts`

File: front/src/Utils/BrowserUtils.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        const currentBrowser = this.getBrowser();
+        const foundBrowser = this.supportedBrowsers.find(
+            (supportedBrowser) => supportedBrowser.name === currentBrowser.name
+        );
+
+        if (!foundBrowser) {
+            console.warn(`Unknown browser: ${currentBrowser.name}\\${currentBrowser.version}`);
+        }
+
+        return !foundBrowser || foundBrowser.minVersion <= currentBrowser.version;
```
Comment (gparant) [inline front/src/Utils/BrowserUtils.ts:83]: Why do you not searh `minVersion` directly in the find and `return foundBrowser !== null`?

File: front/src/Utils/BrowserUtils.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        },
+    ];
+
+    public static getBrowser(): Browser {
+        const userAgent = navigator.userAgent;
+        const matches = userAgent.match(/(opera|chrome|safari|firefox(?=\/))\/?\s*(\d+)/i) || [];
+
+        // Is Opera
+        if (matches[1]?.toLowerCase() === "chrome") {
+            const matchVersion = userAgent.match(/\bOPR\/(\d+)/);
```
Comment (gparant) [inline front/src/Utils/BrowserUtils.ts:41]: Could you not create one regex to catch `userAgent` and `version`?

**Pull request closed without merging**